### PR TITLE
preventing tap even on drag for dialogList menu

### PR DESCRIPTION
**Pull request by ozFri**
Added a snippet to prevent click on drag, tested on my phone and it seems to work.
closing #102

### CODE REVIEW

**Comment by avirankatzsofi** [comment]
I'm getting the following error when trying to open a dialog:

`Uncaught ReferenceError: touchmoved is not defined`. This comes from `ui-dialoglist.js`, line 2

File: html/ui-dialoglist.js [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
-    text = text.replace(` ${count}`, " " + (count - oldCount) + "");
-    jQuery("span#dialog-btn-label").text(text);
-    jQuery("span#dialog-btn-label").attr("count", count - oldCount);
-  } else {
-    text = text.replace(` ${count}`, "");
-    jQuery("span#dialog-btn-label").text(text);
-    jQuery("span#dialog-btn-label").attr("count", 0);
-  }
+jQuery(document).on("click touch touchend", "span.myDialogs", function(event) {
+  if (touchmoved != true) {
```
Comment (avirankatzsofi) [inline html/ui-dialoglist.js:2]: Let's avoid the big `if`, using

```javascript
if (touchmoved) {
  return;
}
// the rest of the code
```

Comment (ozFri) [reply html/ui-dialoglist.js]: Done

**Comment by avirankatzsofi** [comment]
@ozFri I see that this works on mobile, but not on dekstop. 

If I drag the mouse over `span.myDialogs` on my PC, a dialog popup appears.

File: html/ui-dialoglist.js [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
+    } else {
+      text = text.replace(` ${count}`, "");
+      jQuery("span#dialog-btn-label").text(text);
+      jQuery("span#dialog-btn-label").attr("count", 0);
+    }
+
+    currentDialog = myDialogMenu[event.currentTarget.id];
+    dpt.getDialog(currentDialog.dialog);
+    event.stopImmediatePropagation();
+    event.preventDefault();
```
Comment (avirankatzsofi) [inline html/ui-dialoglist.js:33]: Please note that this is more indented than the `if` block above. Could you fix this?

File: html/ui-dialoglist.js [hunk 3]
```diff
@@ -1,31 +1,40 @@
...
-    jQuery("span#dialog-btn-label").text(text);
-    jQuery("span#dialog-btn-label").attr("count", count - oldCount);
-  } else {
-    text = text.replace(` ${count}`, "");
-    jQuery("span#dialog-btn-label").text(text);
-    jQuery("span#dialog-btn-label").attr("count", 0);
+var touchmoved;
+jQuery(document).on("click touch touchend", "span.myDialogs", function(event) {
+  if (touchmoved) {
+      return;
```
Comment (avirankatzsofi) [inline html/ui-dialoglist.js:4]: I see that the indentation size is 2, but this line uses 4 spaces. Could you fix this?

File: html/ui-dialoglist.js (hunk 2 above)
Comment (ozFri) [reply html/ui-dialoglist.js]: Set file indentation to 2 spaces consistently

File: html/ui-dialoglist.js (hunk 3 above)
Comment (ozFri) [reply html/ui-dialoglist.js]: Set file indentation to 2 spaces consistently

**Comment by ozFri** [comment]
@avirankatzsofi about this not working on desktop, works on my laptop on Chrome and Firefox, I need to try with a real mouse instead of touchpad.

**Comment by avirankatzsofi** [comment]
I'll be more specific:
When I select some text from the dialog list, the dialog itself appears.

I found this, which might come in handy:
https://stackoverflow.com/questions/31982407/prevent-onclick-event-when-selecting-text

**Pull request merged**